ludwig/models/model.py

```diff
@@ -1041,7 +1041,7 @@ def update_output_stats_batch(
         output_features = self.hyperparameters['output_features']
         combined_correct_predictions = None
 
-        for i, output_feature in enumerate(output_features):
+        for _, output_feature in enumerate(output_features):
```

Comment:
this one I already solved in master I believe, you can also remove the enumarate and remove the _,